test/src/test/java/org/corfudb/runtime/view/ViewsGarbageCollectorTest.java

```diff
@@ -0,0 +1,63 @@
+package org.corfudb.runtime.view;
+
+import org.corfudb.protocols.wireprotocol.Token;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.MultiCheckpointWriter;
+import org.corfudb.runtime.ViewsGarbageCollector;
+import org.corfudb.runtime.collections.CorfuTable;
+
+import org.ehcache.sizeof.SizeOf;
+import org.junit.Test;
+
+import java.time.Duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Created by maithem on 11/16/18.
+ */
+
+public class ViewsGarbageCollectorTest extends AbstractViewTest {
```

Comment:
please add a javadoc comment here to illustrate how gc is tested against memory leak.